Sources/Journify/Startup.swift

```diff
@@ -12,6 +12,9 @@ extension Journify: Subscriber {
     internal func platformStartup() {
         add(plugin: JournifyLog())
         add(plugin: StartupQueue())
+        if #available(iOS 14, *) {
```

Comment:
question: still users with older iOS version can install the app, just this part will be skipped?

Reply:
@amineck That’s right! Since IDFA requires a minimum iOS version of 14, we just skip the plugin on older versions.